src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -292,6 +308,125 @@ type BikeRentalStation implements Node & PlaceInterface {
+    """
+    operative: Boolean
+}
+
+"""
+Vehicle rental station represents a location where users can rent bicycles etc. for a fee.
+"""
+type VehicleRentalStation implements Node & PlaceInterface {
+    """
+    Global object ID provided by Relay. This value can be used to refetch this object using **node** query.
+    """
+    id: ID!
+
+    """ID of the vehicle in the format of network:id"""
+    stationId: String
+
+    """Name of the vehicle rental station"""
+    name: String!
+
+    """
+    Number of vehicles currently available on the rental station.
+    See field `allowPickupNow` to know if is currently possible to pick up a vehicle.
+    """
+    vehiclesAvailable: Int
+
+    """
+    Number of free spaces currently available on the rental station.
+    Note that this value being 0 does not necessarily indicate that vehicles cannot be returned
+    to this station, as for example it might be possible to leave the vehicle in the vicinity of
+    the rental station, even if the vehicle racks don't have any spaces available.
+    See field `allowDropoffNow` to know if is currently possible to return a vehicle.
+    """
+    spacesAvailable: Int
+
+    """
+    If true, values of `vehiclesAvailable` and `spacesAvailable` are updated from a
+    real-time source. If false, values of `vehiclesAvailable` and `spacesAvailable`
+    are always the total capacity divided by two.
+    """
+    realtime: Boolean
+
+    """
+    If true, vehicles can be returned to this station if the station has spaces available
+    or allows overloading.
+    """
+    allowDropoff: Boolean
+
+    """
+    If true, vehicles can be picked up from this station if the station has vehicles available.
+    """
+    allowPickup: Boolean
+
+    """If true, vehicles can be currently returned to this station."""
+    allowDropoffNow: Boolean
+
+    """If true, vehicles can be currently picked up from this station."""
+    allowPickupNow: Boolean
+
+    """ID of the rental network."""
+    network: String
+
+    """Longitude of the vehicle rental station (WGS 84)"""
+    lon: Float
+
+    """Latitude of the vehicle rental station (WGS 84)"""
+    lat: Float
+
+    """Nominal capacity (number of racks) of the rental station."""
+    capacity: Int
+
+    """If true, vehicles can be returned even if spacesAvailable is zero or vehicles > capacity."""
+    allowOverloading: Boolean
+
+    """Platform-specific URLs to begin renting a vehicle from this station."""
+    rentalUris: VehicleRentalUris
+
+    """
+    If true, station is on and in service.
+    """
+    operative: Boolean
+}
+
+"""
+Rental vehicle represents a vehicle that belongs to a rental network.
+"""
+type RentalVehicle implements Node & PlaceInterface {
+    """
+    Global object ID provided by Relay. This value can be used to refetch this object using **node** query.
+    """
+    id: ID!
+
+    """ID of the vehicle in the format of network:id"""
+    vehicleId: String
```

Comment:
Should this id have a more generic term that could be shared between RentalVehicle and VehicleRentalStation?